app/src/main/res/menu/activity_app_bar_drawer.xml

```diff
@@ -16,13 +16,7 @@
             android:title="Display Lyrics"
             app:actionLayout="@layout/switch_item"
             app:showAsAction="always" />
-        <item
-            android:id="@+id/song_change_notifications"
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:title="Song Change Notifications "
-            app:actionLayout="@layout/switch_item"
-            app:showAsAction="always" />
+
```

Comment:
Good call, we can always add it back once we're ready to add the functionality for it!